website/docs/r/quicksight_group.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "aws"
+page_title: "AWS: aws_quicksight_group"
+sidebar_current: "docs-aws-resource-quicksight-group"
+description: |-
+  Manages a Resource Quick Sight Group.
+---
+
+# aws_quicksight_group
+
+Resource for managing Quick Sight Group
+
+## Example Usage
+
+```hcl
+resource "aws_quicksight_group" "example" {
+	aws_account_id 	= "123456789123"
```

Comment:
When this argument is made optional, let's omit this here as well. 👍

Reply:
fixed.